### ISSUE TITLE
friends_with? returns true if users are not friends

### ISSUE THREAD

**Issue by tr0ythex** [sungwoncho/has_friendship#21]
User1 sends request to user2
`user1.friend_request(user2)`
And now even if user2 hasn't accepted friend request from user1, **friends_with?** return true
```
user1.friends_with?(user2)
=> true
```
```
user2.friends_with?(user1)
=> true
```

**Comment by sungwoncho** [comment]
Fixed in 0.1.2

**Comment by tr0ythex** [comment]
Thank you man)
